santorini-common/src/objects/tower.rs

```diff
@@ -52,3 +52,19 @@ impl Default for Level {
         Level::Ground
     }
 }
+
+impl fmt::Display for Level {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", *self as usize)
+    }
+}
+
+impl Level {
+    pub fn up(self) -> Level {
+        match self {
+            Level::Ground => Level::One,
+            Level::One => Level::Two,
+            Level::Two | Level::Three => Level::Three,
```

Comment:
Odd behavior here. Maybe this should be a fallible function and error in the case of upping Level::Three.